freshmaker/config.py

```diff
@@ -125,7 +141,7 @@ class Config(object):
             'desc': 'Debug mode'},
         'log_backend': {
             'type': str,
-            'default': None,
+            'default': 'journal',
```

Comment:
What do you think about this @qixiang , I guess it should stay `None` rather than `journal`?

Reply:
right, keep it as `None` is better.